.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.0
+jobs:
+  build:
+    docker:
+      - image: membrane/bionic-membrane:latest
```

Comment:
I would not use membrane image here as this package can be used without membrane. Use the regular `circleci/elixir` image.